.github/ISSUE_TEMPLATE/new-content-issue-template.md

```diff
@@ -0,0 +1,23 @@
+---
+name: New content issue template
+about: 'Use this template when suggesting new handbook content '
+title: ''
+labels: content
+assignees: ''
+
+---
+
+Issue type: NEW CONTENT
+Parent category: **Where should this content be placed?**
```

Comment:
What is this? I think it needs elaboration, e.g. a list of valid parents or a link to documentation.

Reply:
New text:

`Parent category: /category/subcategory (where in the table of contents should this content be placed)`

Not perfect. Let's clarify this further on.

Reply:
Much better though! I'm satisfied with this for now.